pkg/hs/urbit-king/lib/Urbit/King/Main.hs

```diff
@@ -395,7 +396,12 @@ testPill pax showPil showSeq = do
   pill <- fromNounErr pillNoun & either (throwIO . uncurry ParseErr) pure
 
   logInfo "Using pill to generate boot sequence."
-  bootSeq <- genBootSeq (Ship 0) pill False (Fake (Ship 0))
+  bootSeq <- genBootSeq
```

Comment:
Questions for @joemfb re urth/mars:
- Given that the boot sequence has changed in king/serf, what (if any) change needs to be made in the urth/mars "boot message"
- does new mars implement this new boot sequence correctly?

Reply:
The mars/urth interface actually has affordances for this kind of pattern. It should be straightforward to port these changes.